### PR TITLE
[Release Notes Automation] Set the default time zone to PST

### PR DESCRIPTION
**Pull request by yangyansong-adbe**
List some alternative time zones for use in scripts.
"America/New_York" - Eastern Time
"America/Chicago" - Central Time
"America/Denver" - Mountain Time
"America/Los_Angeles" - Pacific Time

@praveek @emdobrin  let me know if you think we should use another time zone in scripts. For now, I have picked central time as the default time zone.

### CODE REVIEW

**Comment by calebk1** [comment]
I'm not Emilia, but I'd recommend either EST or PST, since those generally are the time zones that are the most popular to work off of (unless you're doing UTC).

**Comment by praveek** [comment]
@yangyansong-adbe I prefer changing it to PST.

**Comment by yangyansong-adbe** [comment]
@praveek @calebk1 now the default time zone is PST.

File: .github/scripts/release_notes/index.js [hunk 1]
```diff
@@ -21,6 +21,15 @@ if (token == undefined) {
     throw new Error("token is undefined")
 }
 
+//before running the script, make sure the default time zone is set to PST in the GitHub action
+process.env.TZ = "America/Los_Angeles"
+
+const offset = new Date().getTimezoneOffset()
+
+if (offset != 420) {
+    throw new Error("The default time zone is not set to the Central Time")
```
Comment (praveek) [inline .github/scripts/release_notes/index.js:30]: Can you update the error message?

**Review by praveek — APPROVED** [review]

**Review by emdobrin — APPROVED** [review]
:shipit:

**Pull request merged**